src/plugins/recordTypes/work/columns.js

```diff
@@ -0,0 +1,27 @@
+import { defineMessages } from 'react-intl';
+
+export default (configContext) => {
+  const {
+    formatRefName,
+  } = configContext.formatHelpers;
+
+  return {
+    default: {
+      termStatus: {
+        disabled: true,
+      },
+      creator: {
+        formatValue: formatRefName,
+        messages: defineMessages({
+          label: {
+            id: 'column.works.default.creator',
+            defaultMessage: 'Creator',
+          },
+        }),
+        order: 30,
+        sortBy: 'works_common:workTermGroupList/0/creator',
```

Comment:
Look at the path to the field in fields.js to see what this should be. So `works_common:creatorGroupList/0/creator`.  Test if you have it configured correctly by doing a search, and seeing if clicking on the column header results in the rows being sorted as expected.